### PR TITLE
Add some information and links, to have some context..

### PR DESCRIPTION
**Pull request by strk**
This is to have a bit more than "website" on gitea.io

### CODE REVIEW

**Comment by tboerger** [comment]
Oh sorry, I have restructured it where i move the template to the theme. Please update this PR.

**Comment by strk** [comment]
On Thu, Nov 10, 2016 at 02:23:57AM -0800, Thomas Boerger wrote:

> Oh sorry, I have restructured it where i move the template to the theme. Please update this PR.

`hugo server` fail against master:

Error: Unable to find theme Directory: /home/strk/go/src/github.com/go-gitea/src/website/themes/gitea

did you miss to push something ?

**Comment by tboerger** [comment]
No, it's a separate repo, clone go-gitea/theme into themes/gitea. That's what drone does automatically. I think I will add a make task for that later on.

**Comment by tboerger** [comment]
And look at content/index.md, there you can add this content.

**Comment by strk** [comment]
On Thu, Nov 10, 2016 at 02:33:21AM -0800, Thomas Boerger wrote:

> No, it's a separate repo, clone go-gitea/theme into themes/gitea. That's what drone does automatically. I think I will add a make task for that later on.

Could we register go-gitea/theme as a themes/gitea submodule in website repo ?

**Comment by strk** [comment]
Rebased, content moved where it belongs, separate PR for the submodule is #2

File: content/index.md [hunk 1]
```diff
@@ -6,4 +6,16 @@ weight: 10
 type: "index"
 ---
 
-# Website
+# Gitea - git with a cup of tea
```
Comment (tboerger) [inline content/index.md]: `Gitea - Git with a cup of tea`

File: content/index.md [hunk 2]
```diff
@@ -6,4 +6,16 @@ weight: 10
...
+
+Gitea is a community managed fork of [Gogs](http://gogs.io),
+lightweight code hosting solution written in [Go](https://golang.org/)
+and published under the [MIT](https://github.com/gogits/gogs/blob/master/LICENSE)
+license.
+
+Code of Gitea is currently hosted on [GitHub](https://github.com/go-gitea/)
+but we hope in the future to self-hosted on a Gitea instance.
```
Comment (tboerger) [inline content/index.md]: Scratch that.

**Review by tboerger — CHANGES_REQUESTED** [review]

File: content/index.md (hunk 2 above)
Comment (strk) [reply content/index.md]: Scratch what, the hope line ?

Comment (tboerger) [reply content/index.md]: Yes

**Comment by strk** [comment]
Done, and rebased

**Comment by lunny** [comment]
LGTM

**Comment by thibaultmeyer** [comment]
LGTM

**Comment by strk** [comment]
@tboerger we need you acceptance of requested changes (but I have the option of `Dismiss review`, looks like, I'll refrain to exercise it)

**Review by tboerger — APPROVED** [review]

**Comment by tboerger** [comment]
LGTM

**Comment by strk** [comment]
continuous-integration/drone/push still blocks this, what is it for ?

**Comment by tboerger** [comment]
The build have been fine, idk. Please do a force push to trigger a rebuild

**Comment by tboerger** [comment]
We need to disable the requirement for the push part within protected branches of I see that correctly

**Comment by strk** [comment]
Can you try disabling it so we can move forward ?

**Pull request merged**